dissect/target/plugins/os/windows/wer.py

```diff
@@ -12,6 +12,21 @@
 from dissect.target.target import Target
 
 camel_case_patterns = [re.compile(r"(\S)([A-Z][a-z]+)"), re.compile(r"([a-z0-9])([A-Z])"), re.compile(r"(\w)[.\s](\w)")]
+arraylike_pattern = re.compile(r"([_a-zA-Z][a-zA-Z0-9]*)\[(\d+)\]")
+
+
+def _clean_key(key: str) -> str:
```

Comment:
```suggestion
@functools.lru_cache()
def _clean_key(key: str) -> str:
```
As it seems most of the keys are shared between .WER files and this will speed things up a bit, as @yunzheng mentioned.